index.js

```diff
@@ -1,128 +1,154 @@
-function restartGame(){
-    window.location.reload()
+
+function restartGame() {
+    window.location.reload();
+}
+let counter = 0;
+const timer = document.querySelector("#timer");
+//increasing the counter
+function inter() {
+  counter++;
+  console.log();
+  timer.innerHTML = "<b>" + counter + "</b>";
+}
+var interval;
+function setTimer() {
+  interval = setInterval(inter, 1000);
+}
+
+// open model when click on start game button
+
+function openModel() {
+  counter = 0;
+  clearInterval(interval);
+  timer.innerHTML = "<b>" + 0 + "</b>";
+  const box = document.getElementById("model");
+  const prevent_box = document.getElementById("prevent-box");

```

Comment:
these you you are using in close modal also , so better to declare them on the top one time